recOrder/scripts/cli.py

```diff
@@ -174,7 +197,18 @@ def cli():
     type=click.Choice(["position", "channel", "p", "c"]),
     help="Layers as 'position' ('p') or 'channel' ('c')",
 )
-def view(filename, positions=None, layers=None):
+@click.option(
+    "--overlay",
+    "-o",
+    is_flag=True,
+    default=False,
+    help="""
+    Computes a retardance-orientation HSV overlay layers. Requires channel 
+    names `Retardance` and `Orientation` to be present, and requires `--layers 
+    position`
+    """,
+)
+def view(filename, positions=None, layers=None, overlay=False):
```

Comment:
My impression is that default value of arguments in the signature are not need in click options if the decorators already have them (and they are different here).

Reply:
Thanks!